local_test.go

```diff
@@ -35,3 +37,28 @@ func TestDial_IPV6(t *testing.T) {
 		})
 	}
 }
+
+func TestSendReceive(t *testing.T) {
+	c, err := amqp.Dial("amqp://")
+	require.NoError(t, err)
+	defer c.Close()
+
+	ssn, err := c.NewSession()
+	require.NoError(t, err)
+
+	r, err := ssn.NewReceiver(amqp.LinkAddressDynamic())
+	require.NoError(t, err)
+	var m *amqp.Message
+	done := make(chan error)
+	go func() {
+		var err error
+		defer func() { done <- err; close(done) }()
+		m, err = r.Receive(context.Background())
+		m.Accept()
```

Comment:
This shouldn't be called if `err == nil`

Reply:
Quite right